uiowa/uiowa
# Enable and configure ultimate_cron.

Resolves #3059 

### Reminder
Cron runs once an hour in prod, offset by 10 minutes for each application.

Cron runs once every 3 hours in dev/test, offset by 10 minutes for each application.

### Changes
Enables ultimate_cron on all sites and configures all jobs to run hourly except Acquia Connector, Node Revision Delete and Tmp File Cleanup. Acquia Connector runs nightly at midnight, NRD runs nightly every hour from midnight to 2am and Tmp File Cleanup runs every 3 hours.

Still investigating cron logging to normal channels. This seems to use the DB to log... 👎🏻 

### Testing
- Go to https://uiowa.dev.drupal.uiowa.edu/admin/config/system/cron/jobs
- Review cron schedules, last run time, status, etc.
  - Note that I did not merge the latest NRD schedule change into dev. Its still set at 1-5 which I think is overkill.
- Run cron.
- See that AC and NRD do not run and last ran overnight.

I was trying to review stack metrics in dev but its not working.
